README.md

```diff
@@ -1 +1,52 @@
-# crypto-market-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Crypto Market App
+
+> Crypto Market is a SPA that allow users to check live prices of cryptocurrencies. Built with React and CoinGecko API.
```

Comment:
- [ ] Kindly deploy your app and include the link in your README.

- [ ] Kindly record a video presentation about the project no longer than 5 minutes and provide the link. Remember also to highlight the piece of code you are proud of making.